spec/targeted_refresh/processor_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'topological_inventory/ansible_tower/targeted_refresh/processor'
+
+RSpec.describe TopologicalInventory::AnsibleTower::TargetedRefresh::Processor do
+  describe "#self.process!" do
+    let(:message) { double(:message => 'SomeModel.some_method') }
+    let(:payload) { double("Payload") }
+
+    it "expects message in Model.Method format" do
+      processor = double(:process => nil)
+
+      expect(described_class).to receive(:new)
+        .with('SomeModel', 'some_method', payload)
+        .and_return(processor)
+
+      described_class.process!(message, payload)
+    end
+  end
+
+  describe "#process" do
+    context "on implemented operation" do
+      let(:job_refs) { %w[10 20] }
+      let(:message) do
+        double(:message => 'ServiceInstance.refresh',
+               :payload => {
+                 :source_id  => '1',
+                 :source_uid => '3bfb8667-2b00-480b-bcbf-452bfb34a440',
+                 :params     => [{:task_id => '1', :source_ref => job_refs[0], :request_context => {"x-rh-identity": "xxx", "x-rh-insights-request-id": "111"}},
+                                 {:task_id => '2', :source_ref => job_refs[1], :request_context => {"x-rh-identity": "yyy", "x-rh-insights-request-id": "222"}}]
```

Comment:
Rubocop pointed out these two lines for "Use hash rockets" which I would recommend resolving.  These lines are using mixed syntax which we should avoid.